recipes/plasmapy/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{% set name = "plasmapy" %}
+{% set version = "0.1.dev968" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "f127782615810e6f45f39f63a51b7eda76a9fe0286b4aef54f265cc744e946a6" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.{{ file_ext }}'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  script: python setup.py install --offline --single-version-externally-managed --record=record.txt
+  skip: True  # [py2k]
+  skip: True  # [py35]
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - numpy
+    - scipy
+    - astropy
+    - coveralls
```

Comment:
this is usually not a requirement of a package, isn't it?

Reply:
No, you're right, I think I put that in to try and solve some problem and then forgot to take it out when it didn't work.